specs/DesignDiscussions/StackedImageSet.md

```diff
@@ -0,0 +1,34 @@
+# Stacked Images/Badging
+### Problem 
+Customers are requesting the ability to stack images like so:
+![img](assets/StackedImageSet/stackedImage.png)
+
+This will enable them to show a chat as a group chat in a messenger app.
+#### Key Features 
+*	Stack a maximum of 2 images. The green status indicator would not be displayed.
```

Comment:
Is there a reason for displaying only 2 images? 
3 and more could be displayed in a slightly different fashion (similar to Teams?)

Reply:
No, this is client specific feature. we are going to share this spec with Team to expand the spec further.